src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const successHandler = () => {
+  const successMessage = document.createElement('div');
+
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMessage);
+};
+
+const errorHandler = () => {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+};
+
+promise1
+  .then(successHandler)
+
+  .catch(errorHandler);
```

Comment:
The promise1 is resolved when the logo is clicked, triggering the successHandler. However, if the promise is rejected, the errorHandler will be called, which is not possible in the current setup since promise1 only resolves. Consider removing the catch block for promise1 as it will never be rejected.